### DISCUSSION TITLE
Make it for windows

### DISCUSSION THREAD

**Discussion by MoonDevil1** [vega-org/vega-app#159] [General]
Make it for windows without and emulator

**Comment by himanshu8443** [comment]
I tried, the state of React-Native Windows is bad
Microsoft need to do somthing.

**Reply by UTKARSHKUMAR712** [reply]
done i Have made it you can try https://github.com/UTKARSHKUMAR712/Poly_movies.git

**Reply by Zenda-Cross** [reply]
great work

**Reply by UTKARSHKUMAR712** [reply]
Thanks for your appreciation and For your great work to built the heart of this project.

**Reply by Zenda-Cross** [reply]
You can join Discord and share it if you want https://discord.gg/cr42m6maWy, people there will like it

**Comment by proh7772** [comment]
> Make it for windows without and emulator

React on my post too for make this possible soon.

**Reply by UTKARSHKUMAR712** [reply]
done i Have made it you can try https://github.com/UTKARSHKUMAR712/Poly_movies.git

**Comment by proh7772** [comment]
Utkarsh Kumar, Thank you so much for this effort.
You really made it. You take out time just make for the users. I am really
appreciate it.

You are awesome.

**Comment by proh7772** [comment]
<img width="1832" height="1017" alt="image" src="https://github.com/user-attachments/assets/57171384-5fbe-4f76-b3cb-87b175329f36" />
🙏 Appreciation

> I really appreciate your amazing work — thank you for all the effort you put into this!

✅ Feature Request

> I have one small request that could improve the user experience:

🔗 Add a "Copy Link" Button

> Please add a 📋 Copy Link button next to the ▶️ Play button in the Available Streams section.

🎯 Purpose

> This will allow users to:
> 
> 🎥 Play the stream in an external player
> 
> 🌍 Choose their preferred language easily

**Reply by UTKARSHKUMAR712** [reply]
I’ll definitely implement this—I'd already planned to add support for external players like VLC when user click it will give option to choose and also for direct download and acess downloaded movies directly from app . As for 🌍 choosing the preferred language, that part is unfortunately out of my control. Some providers offer language options, but many do not. Still, I’ll explore what’s possible.

I’m currently busy as I’ve gone home for Diwali, but I’ll share an update once I’m back and have time to work on it.

**Reply by proh7772** [reply]
Okay, have fun on Diwali with your family. We already know that you are doing your best and thanks for that. ♥️

**Reply by UTKARSHKUMAR712** [reply]
**Added extrenal player support plese use vlc for best experience.
Unified Search Flow — performSearch() now supports external triggers, collects provider lanes incrementally, and keeps result rendering in a single view for faster feedback.
